RELEASE.md

```diff
@@ -0,0 +1,86 @@
+# Release process
+
+Start by making a release issue using the template below. The issue checklist can be followed to
+release a new version of the Helm chart and help everybody coordinate. Do some
+copy pasting!
+
+## Issue title: Release x.y.z
+
+This issue will be used to coordinate the next release of the Helm
+chart according to the instructions in [RELEASE.md](RELEASE.md).
+Below is the checklist for this release.
+
+## Pre-release iteration 1
+
+- Make a changelog
+  - [ ] Summarize points of interest.
+  - [ ] List breaking changes and mention the [upgrade instructions](https://z2jh.jupyter.org/en/latest/upgrading.html).
+    - [ ] Update the upgrade instructions.
+  - [ ] List features with brief descriptions.
+
+- Pre-release
+  - [ ] Create and push a git tag
+
+    ```bash
+    git checkout master
+    git pull <REMOTE> master
+    git tag -a x.y.z-beta.1
+    git push <REMOTE> --tags
+    ```
+
+- Update documentation
+  - [ ] Update old version references to the new version
+
+- Communicate
+  - [ ] Write a discourse post
+
+- Verify
+  - [ ] Follow one set of instructions to deploy on z2jh.jupyter.org
+
+## Final release
+
+- Update changelog
+  - [ ] Update the changelog
+  - [ ] Generate and add a list of contributors
+
+    ```bash
+    # install dependencies for the script
+    pip install pygithub requests-cache tqdm
+
```

Comment:
Add something about bumping the dates?
https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/83f776469bc53150b2e577212ec63a17455bb4e9/tools/contributors.py#L85-L94